additionalResolvers/course.ts

```diff
@@ -5,6 +5,7 @@ const resolvers: Resolvers = {
         // combines suggestions from the chapters of the course to a single list of suggestions for the
         // whole course
         suggestions: {
+            // HINT use course service filters instead of manually filtering the chapters
```

Comment:
What exactly do you mean with "Hint"? Is it a "TODO"? Because then it should be written as such IMO (TODOs are usually highlighted in the IDE and can also be searched for if you use standardized terminology across the whole project)

Reply:
HINT is also a standard term in the java world. didnt know it wasnt everywhere

Reply:
It's like a non urgent todo

Reply:
ok maybe i just didn't know about it